### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by moonbear46**
Hey, I would like to offer you an almost complete translation of Skoice in my native language. I will be happy to make the missing translations with an explanation of your color code. I am at your disposal for any question.

### CODE REVIEW

**Review by LucasCdry — CHANGES_REQUESTED** [review]
Thanks for the hard work! We really appreciate it.

Here are some issues:
- Please stick to German capitalizing rules for titles (only the first letter, if I am right).
- Try to be consistent as much as possible (`Nähe Voice-Chat`, `Voice-Chat in der Nähe`, `Proximity Voice Chat`, `Proximity-Voice-Chat`).
- Try to avoid English words as much as possible (for instance, in French, we translated `Lobby` into the French for `Main voice channel`).
- Address some requested changes to align with a recent commit I have made.

Concerning the color code, here is the rules:
- Gray is our base color (&7), no need to put it at the beginning of messages though.
- Anything between quotation marks is colored in yellow (`&e`).
Emphasized words are either green (`&a`, for an intended behaviour) or red (`&c`, for an unexpected behaviour). I don't know anything about German grammar, but for English and French, the coloured part is:
  - the verb *or* the `-ed` form of the verb if there is one ("Your Minecraft account is __already **linked**__ to Discord.")
  - the adverb if there is one ("Your Minecraft account is __**already** linked__ to Discord.")
  - the direct object that follows the verb ("You have __not provided **a valid code**__.")
  - "not" when there is a negation ("You have __**not** provided a valid code__.")

If those rules cannot be applied to the German language, do not forget our keyword: **consistency**!

Thanks again for the effort you have put in!

File: src/main/resources/lang/DE.yml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+            alternative-description: "Horizontaler Radius: %s Blöcke - Vertikaler Radius: %s Blöcke"
+    horizontal-radius:
+      title: "Horizontaler Radius"
+      description: "Dies ist der maximale horizontale Abstand zwischen den Spielern."
+    vertical-radius:
+      title: "Vertikaler Radius"
+      description: "Dies ist der maximale vertikale Abstand zwischen den Spielern."
+    advanced-settings:
+      title: "Erweiterte Einstellungen"
+      description: "Manage other parameters."
```
Comment (LucasCdry) [inline src/main/resources/lang/DE.yml:122]: Needs to be translated

File: src/main/resources/lang/DE.yml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      description: "Wählen Sie diesen Modus, wenn Sie Spielmodi spielen möchten, die nur einen begrenzten Bereich (kürzere Entfernungen) erfordern."
+    customize:
+      title: "Anpassen"
+      description: "Stellen Sie die Entfernungen nach Ihren persönlichen Vorlieben ein."
+    enter-a-value:
+      title: "Einen Wert eingeben"
+      description: "Diese Einstellung ist derzeit auf %s Blöcke eingestellt. Der Wert muss zwischen 1 und 1000 liegen."
+    changelog:
+      title: "Skoice 2.1"
+      description: "**General:**\n• Discover a whole new and intuitive configuration interface.\n• Let us choose the best distances for your personal use of Skoice in the \"Mode\" menu.\n• Link your accounts effortlessly thanks to our new linking process.\n• Skoice now uses slash commands.\n\n**New Features:**\n• Action bar alert\n• Channel visibility toggle\n• \"/invite\" command in Discord\n• French translation"
```
[the remark this answers is not in the document]
Comment (moonbear46) [reply src/main/resources/lang/DE.yml:181]: Hey, how can I suggest my modifications to this?

**Comment by moonbear46** [comment]
Hey Lucas! I corrected according to your feedback and finished all the translations thanks to your explanation of the color code. Feel free to give me more feedback!

**Review by LucasCdry — CHANGES_REQUESTED** [review]
Hey! Just a few missing translations to align with my latest commits. I will merge your work as soon as those have been made.

**Pull request merged**